README.md

```diff
@@ -16,20 +16,31 @@ with [Anthos Service Mesh] features enabled.
     ```bash
     # [PACKAGE] is any directory name under the repository root.
     # [OUTPUT_DIR] refers to the desired local location of the package.
-    $ kpt pkg get
-    https://github.com/GoogleCloudPlatform/anthos-service-mesh-packages.git/[PACKAGE] [OUTPUT_DIR]
+    $ kpt pkg get https://github.com/GoogleCloudPlatform/anthos-service-mesh-packages.git/[PACKAGE] [OUTPUT_DIR]
+
+    # e.g. to get the asm package into the current folder, run the following,
+    # which will create an 'asm' folder under the current folder
+    $ kpt pkg get https://github.com/GoogleCloudPlatform/anthos-service-mesh-packages.git/asm .
     ```
 4.  Verify all configurable values are as desired:
 
     ```bash
-    $ kpt cfg list-setters [OUTPUT_DIR]
+    $ kpt cfg list-setters [ASM_FOLDER]
```

Comment:
I explained what `PACKAGE` and `OUTPUT_DIR` were above, so if adding `ASM_FOLDER` I think you should describe it to be explicit. Or just rename `OUTPUT_DIR` above.